ui/src/main/webapp/src/app/services/application-group.service.ts

```diff
@@ -52,6 +53,17 @@ export class ApplicationGroupService extends AbstractService {
             .catch(this.handleError);
     }
 
+    getPackageMetadata(id:number): Observable<PackageMetadata> {
+        let headers = new Headers();
+        let options = new RequestOptions({ headers: headers });
+
+        let url = this.PACKAGE_METADATA.replace(/#\{groupID\}/, id.toString());
```

Comment:
Is it necessary to use regular expression here? I think 
```
this.PACKAGE_METADATA.replace("#{groupID}", id.toString());
```
should be sufficient and maybe even faster.